components/groups/components/Proposals.tsx

```
@@ -0,0 +1,247 @@
+          {!tally ? <span className="loading loading-spinner loading-xs"></span> : status}
+
+          {
+            // Show vote modal if the proposal is selected, cannot be outside `<tr>`.
+            showVoteModal && (
+              <VoteDetailsModal
+                policyAddress={policyAddress}
+                proposalId={proposal.id}
+                onClose={() => {
+                  setShowVoteModal(false);
+                  const query = { ...router.query };
+                  delete query.proposalId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)